atef/widgets/config/data_active.py

```diff
@@ -874,8 +874,9 @@ def attr_to_target(self, attr: OphydAttributeData) -> Target:
         dot_attr = attr.attr
         _attr = '_' + dot_attr.replace('.', '_')
         dev_name = full_name[:-len(_attr)]
+        write_pv = getattr(attr.signal, 'setpoint_pvname', attr.pvname)
```

Comment:
Is it prohibitive to include the `setpoint_pvname` as part of `OphydAttributeData`?
https://github.com/pcdshub/atef/blob/ed137620cfd8f411e16fd27a68d806708b61512e/atef/widgets/ophyd.py#L67-L76

Reply:
This isn't necessary to merge the PR but it's something I noticed because it seems weird to need to dig into the `attr.signal` at this stage.

Reply:
It might take a bit of refactoring but it's probably clearer moving forward.  The existing ophyd device model does a similar access and we'd just need to clean that up